coil/src/androidTest/java/dev/chrisbanes/accompanist/coil/CoilTest.kt

```diff
@@ -75,8 +76,9 @@ import java.util.concurrent.atomic.AtomicInteger
 @LargeTest
 @RunWith(JUnit4::class)
 class CoilTest {
+    @Suppress("DEPRECATION") // createAndroidComposeRuleLegacy
     @get:Rule
-    val composeTestRule = createComposeRule()
+    val composeTestRule = androidx.compose.ui.test.junit4.createAndroidComposeRuleLegacy<ComponentActivity>()
```

Comment:
Add a comment why legacy is needed?

Reply:
Not sure it's worth it. I hope to get the new stuff landed in #186. I just need to unblocked this PR.